.travis.yml

```diff
@@ -98,21 +98,26 @@ jobs:
       local_dir: upload
       on:
         tags: true
-  - stage: Build docker (dev)
+  - stage: Build docker (Dev/Nightly)
+    if: tag IS blank
     install: true
-    env:
-    - OWNER=streamr
-    - IMAGE_NAME=broker-node
-    - TAG=dev
     script:
-    - docker build -t $OWNER/$IMAGE_NAME:$TAG --build-arg NPM_TOKEN=${NPM_TOKEN} .
```

Comment:
What changed for you to drop --build-arg?

Reply:
We haven't been using it for a while in Dockerfile. It is a remnant of the time `streamr-network` (or as it was known back then `@streamr/network`) was a private NPM package.